src/main/js/controls/Gantt/helpers/actionHelpers.js

```diff
@@ -272,11 +275,11 @@ const drawActionDataPoints = (scale, config, canvasSVG) =>
  * @param {object} graphContext - Gantt instance
  * @param {object} trackPathSVG - Track container element
  * @param {object} trackLabel - Track label
- * @param {object} actions - input JSON for creating an action
+ * @param {object} gantt - input config for creating an action
```

Comment:
This was updated because I also needed access to `gantt.actionKeys`.

Regarding the Gantt chart. I have added four more objects to the Gantt config namely `actionKeys`, `activitykeys`, `eventKeys` and `taskKeys`. To keep a count and name of all the keys currently shown in the graph.